### DISCUSSION TITLE
stac for internal use

### DISCUSSION THREAD

**Discussion by wiesehahn** [radiantearth/stac-spec#1194] [General STAC Q&A]
Let's say we have a lot of geographic data (aerial imagery, point clouds, vector geometries,...) which is primarily meant for internal use at the organization. I could still imagine a benefit of using stac-spec to catalogue the data and e.g. make it searchable with stac browser etc. (Also to have it conformant if data can be published at some point)
Does it make sense to use stac-spec for local files or am I missing something?

**Comment by m-mohr** [comment]
I think it could make sense. To be able to use STAC Browser, you'd need to host them via HTTP(S) though, which of course can be on an internal host. For searching (and not just browsing) abilities, you'd need to set up an API instance anyway though.

**Comment by chiarch84** [comment]
In our orgnisation (Joint Research Center of the EU commission) we do exactly what you described. We have data on an internal distributed file system. We catalogued it all through STAC and are using STAC browser and API for discovery and searching. In the href of the assets instead than having an http url we have a file:// path

**Reply by wiesehahn** [reply]
nice, this sounds good and my goal is to implement it here too. But unfortunately the geodata management here (federal research institute) is not in a good shape until now. If you have any information about your path towards STAC adoption or the geodata management in general I would be happy to learn.

**Reply by chiarch84** [reply]
Well I think it really depends on the original situation that you have. In our case for example we had 0 metadata for collections and had items in OGC format in a JSONB column of Postgis. We moved everything to Elasticsearch and created 1 index for collections and more than 1 indexes for items (but all item indexes have in common the main stac mandatory fields). On top of it we're using the STAC Browser and the STAC FAST API for Elastic so we didn't have to write too much code to have our own configuration. Don't know if you have some more specific questions though.
